opensearch/src/test/java/org/opensearch/sql/opensearch/data/value/OpenSearchExprValueFactoryTest.java

```diff
@@ -671,53 +672,53 @@ public void constructIP() {
         tupleValue("{\"ipV\":\"192.168.0.1\"}").get("ipV"));
   }
 
+  private static final double TOLERANCE = 1E-5;
+
   @Test
   public void constructGeoPoint() {
```

Comment:
Could you add some bad cases testing in `constructGeoPointFromUnsupportedFormatShouldThrowException` for new formats? https://github.com/opensearch-project/sql/blob/81ae699cf43c7dead0e96a583243ac3c30987989/opensearch/src/test/java/org/opensearch/sql/opensearch/data/value/OpenSearchExprValueFactoryTest.java#L688

Reply:
done